ios/AppLovinMAXNativeAdLoader.m

```diff
@@ -0,0 +1,140 @@
+#import <React/RCTUIManager.h>
+#import "AppLovinMAX.h"
+#import "AppLovinMAXNativeAdLoader.h"
+
+@interface AppLovinMaxNativeAdLoader ()  <MANativeAdDelegate>
+
+@property (nonatomic, strong, nullable) MANativeAdLoader *nativeAdLoader;
+@property (nonatomic, strong, nullable) MANativeAdView *nativeAdView;
+@property (nonatomic, weak) id<AppLovinMAXNativeAdLoaderDelegate> nativeAdLoaderDelegate;
+
+@end
+
+@implementation AppLovinMaxNativeAdLoader
+
+- (void)destroy
+{
+    [self destroyNativeAdView];
+}
+
+- (void)load:(NSString *)adUnitIdentifier placement:(nullable NSString *)placement customData:(nullable NSString *)customData extraParameters:(nullable NSDictionary *)extraParameters delegate:(id<AppLovinMAXNativeAdLoaderDelegate>)delegate
+{
+    if ( !self.nativeAdLoader )
+    {
+        self.nativeAdLoader = [[MANativeAdLoader alloc] initWithAdUnitIdentifier: adUnitIdentifier
+                                                                             sdk: [AppLovinMAX shared].sdk];
+        self.nativeAdLoader.revenueDelegate = [AppLovinMAX shared];
+        self.nativeAdLoader.nativeAdDelegate = self;
+    }
+    
+    self.nativeAdLoader.placement = placement;
+    self.nativeAdLoader.customData = customData;
+    if ( extraParameters )
+    {
+        for ( NSString *key in extraParameters )
+        {
+            [self.nativeAdLoader setExtraParameterForKey: key value: extraParameters[key]];
+        }
+    }
+    
+    self.nativeAdLoaderDelegate = delegate;
+    
+    [self.nativeAdLoader loadAdIntoAdView: [self createNativeAdView]];
```

Comment:
I still don't understand the hack with `createNativeAdView`. Could you explain in detail?